src/collectors/diskusage/diskusage.py

```diff
@@ -287,4 +287,4 @@ def collect(self):
                 for key in metrics:
                     metric_name = '.'.join([info['device'], key]).replace(
                         '/', '_')
-                    self.publish(metric_name, metrics[key])
+                    self.publish(metric_name, metrics[key], precision=4)
```

Comment:
Yes, it was.

As I said back in September, I believe the problem I had experienced in March is that without setting the `precision` all metrics are rounded to their integer values, which makes a lot of iostat-related values (like `service_time`) useless.